common/membership/ringpop/test_cluster.go

```diff
@@ -165,6 +171,8 @@ func newTestCluster(
 			mockMgr,
 			resolver,
 			2*time.Second,
+			3*time.Second,
+			joinTime,
```

Comment:
optional nit: I think we could improve the readability here with some options (either via the options pattern or a struct)